doc/whats_new/v0.22.rst

```diff
@@ -40,13 +40,19 @@ Changelog
     where 123456 is the *pull request* number, not the issue number.
 
 :mod:`sklearn.decomposition`
-..................
+............................
 
 - |Enhancement| :class:`decomposition.IncrementalPCA` now accepts sparse
   matrices as input, converting them to dense in batches thereby avoiding the
   need to store the entire dense matrix at once.
   :pr:`13960` by :user:`Scott Gigante <scottgigante>`.
 
+- |Enhancement| :class:`decomposition.PCA` and
+  :class:`decomposition.TruncatedSVD` has a new solver `lobpcg` which
+  accelerate computation.
```

Comment:
I think there was also the matter of better accuracy?

Reply:
it's the same thing for iterative methods